.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,12 +47,23 @@ Enter your extended release note in the block below. If the PR requires addition
 
 <!--
 This section can be blank if this pull request does not require a release note.
-Please use the following format for linking documentation or pass the
-section below:
+
+The Kubernetes Release Notes website (https://relnotes.k8s.io) is able to
+provide additional documentation to the end users, which should help them to
+be able to provide additional documentation to the end users about the changes
+in this PR.
+
+If you decide to add links which point to resources within git repositories,
```

Comment:
This sentence isn't clear on the surface. I'm not sure what you're asking of me as a contributor.

Reply:
> Something like:
> 
> ```
> For KEPs or other supporting documentation, please link to a specific commit
> and avoid linking directly to master. See <link> for further guidelines.
> ```

I picked-up the comment of @mrbobbytables and rephrased the paragraph. WDYT?